.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```

Comment:
I would vote to remove this, it's captured above

Reply:
I saw we keep it, nothing is wrong with verbosity and this provides more detail

Reply:
Making people repeat themselves just means you get low quality info

Reply:
I would argue describing a bug and the expected result are separate issues

Reply:
- refuses to elaborate
- leaves